src/Model/Blog.php

```diff
@@ -443,6 +443,9 @@ public function getSettingsFields()
             ]
         );
 
+        $fields->fieldByName('Root.Users')
+                ->setTitle(_t(__CLASS__ . '.Users', 'Users'));
```

Comment:
```suggestion
        $fields->findTab('Root.Users')->setTitle(_t(__CLASS__ . '.Users', 'Users'));
```
No need for this to span two lines. Also please use `findTab()` to, well, find the tab.

Reply:
Copied from `Root.Categorisation`, but now changed it to findTab. And also applied this to `Root.Categorisation` to improve this part aswell.

Reply:
Ahh, if I'd known it was copied I wouldn't have said anything. Updating both is a good move though, thank you.